opa-manager-operator/tests/unit/conftest.py

```diff
@@ -0,0 +1,47 @@
+import unittest.mock as mock
+
+import pytest
+from ops.pebble import ServiceStatus
+from ops.testing import Harness
+
+from charm import OPAManagerCharm
+
+
+@pytest.fixture(autouse=True)
+def lk_client():
+    with mock.patch("ops.manifests.manifest.Client", autospec=True) as mock_lightkube:
+        with mock.patch("charm.Client", mock_lightkube):
+            yield mock_lightkube.return_value
+
+
+@pytest.fixture
+def harness(mocker):
```

Comment:
I think you will need to include [Pytest-mock](https://pypi.org/project/pytest-mock/) in the test dependencies to use the mocker fixture.